packages/tools/devtools/devtools-view/src/components/OpLatencyView.tsx

```diff
@@ -44,6 +51,105 @@ const useStyles = makeStyles({
 export function OpLatencyView(): React.ReactElement {
 	const styles = useStyles();
 
+	const messageRelay = useMessageRelay();
+	const [durationOutboundBatchingData, setDurationOutboundBatchingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Outbound",
+				uuid: "durationOutbound",
+				xAxisDataKey: "outboundtimestamp",
+				yAxisDataKey: "durationOutboundBatching",
+			},
+			data: [],
+		});
+	const [durationNetworkData, setDurationNetworkData] = React.useState<GraphDataSet>({
+		graphType: "line",
+		schema: {
+			displayName: "Duration Network",
+			uuid: "durationNetwork",
+			xAxisDataKey: "networktimestamp",
+			yAxisDataKey: "durationNetwork",
+		},
+		data: [],
+	});
+	const [durationInboundToProcessingData, setDurationInboundToProcessingData] =
+		React.useState<GraphDataSet>({
+			graphType: "line",
+			schema: {
+				displayName: "Duration Inbound",
+				uuid: "durationInbound",
+				xAxisDataKey: "inboundtimestamp",
+				yAxisDataKey: "durationInboundToProcessing",
+			},
+			data: [],
+		});
+
+	React.useEffect(() => {
+		/**
+		 * Handlers for inbound messages.
+		 */
+		const inboundMessageHandlers: InboundHandlers = {
+			[TelemetryEvent.MessageType]: async (untypedMessage) => {
+				const message = untypedMessage as TelemetryEvent.Message;
+				const eventContents = message.data.event.logContent;
+				if (!eventContents.eventName.endsWith("OpRoundtripTime")) {
+					return true;
```

Comment:
Double check the interface docs on this, but I believe the contract is to return `false` when you aren't handling the incoming message. So for any of the early-returns where we aren't actually doing anything with the message, we should return false.

Reply:
@Josmithr  the true/false flag in the interface indicates whether the message was handled or not. In this case, does dismissing the unrelated ops count as handling them?

Reply:
No. Since we're effectively ignoring the incoming message in that case, I believe it's more correct to return false.

FWIW, I believe the returned boolean just changes verbose logging output. So it doesn't matter too much functionally. But semantically, I believe we say false in this case.